zgrab_schema.py

```diff
@@ -820,15 +820,15 @@
                         }),
                         "serial":String(),
                         "cert_type":SubRecord({
-                            "id":Integer(),
+                            "id":Unsigned32BitInteger(),
```

Comment:
I'm not quite sure what this field is, so can't really provide a review on that it's typed correctly. I'm trusting that you checked whatever relevant documentation.